packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -1,21 +1,22 @@
 import { Theme, Typography, Spacing, FontWeightValue, FontSize, FontSizes, Variants } from '@fluentui-react-native/theme-types';
 import { Platform } from 'react-native';
 import { getStockWebPalette, getStockWebDarkPalette } from './defaultColors';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
 
 function _defaultTypography(): Typography {
   const defaultsDict = {
     sizes: {
-      caption: 10 as FontSize,
-      secondary: 12 as FontSize,
-      body: 14 as FontSize,
-      subheader: 16 as FontSize,
-      header: 20 as FontSize,
-      hero: 28 as FontSize,
+      caption: globalTokens.font.size['100'] as FontSize,
+      secondary: globalTokens.font.size['200'] as FontSize,
+      body: globalTokens.font.size['300'] as FontSize,
+      subheader: globalTokens.font.size['400'] as FontSize,
+      header: globalTokens.font.size['500'] as FontSize,
+      hero: globalTokens.font.size['700'] as FontSize,
       heroLarge: 42 as FontSize,
     } as FontSizes,
     weights: {
-      regular: '400' as FontWeightValue,
-      semiBold: '600' as FontWeightValue,
+      regular: globalTokens.font.weight.regular.toString() as FontWeightValue,
+      semiBold: globalTokens.font.weight.semibold.toString() as FontWeightValue,
```

Comment:
why do we need toString?

Reply:
FontWeightValue is string type, and the font weights in the tokens file are numbers.

@TravisSpomer maybe we should change the output of the font weight to strings to have consistency, to get rid of the cast here.

Reply:
Sure; we can change it to whatever's convenient for the platform. The outputs of the "json" folder are just things I threw together as an example; I could make a separate set of React Native outputs that use strings for font weights, and any other customizations you need.

(Sorry for the delay; I don't seem to get email notifications for @mentions to things under the `microsoft` scope. 😒)

Reply:
Thanks! That would be helpful. I'm working on the change to integrate the pipeline into this repo, so if I can just change the input file to output the right thing, I can do that on my end, too.